### PR TITLE
Improve APIError messages

### LINKED ISSUE #1138: Improve APIError messages
Currently, error messages are partially unspecific when creating tables via the API, e.g. for table and column header names

- [x] improve err msg for wrong column headers [e8588e8](https://github.com/OpenEnergyPlatform/oeplatform/commit/e8588e8c89593c36f8a296f4c52d7b1443780edd)
- Same error messages implemented for all three checks to give user the full picture of possible errors, independent of other error types, to avoid piecewise error correction by the user.
- [x] improve err msg for table names

### PR DESCRIPTION
**Pull request by chrwm**
Closes #1138

### CODE REVIEW

**Comment by chrwm** [comment]
I couldn't test on test-OEP due to a lack of access, but tested the changes locally.

**Comment by jh-RLI** [comment]
Great, I can try to have a look at it later today. You tested it via some API request or via the Upload on the Website?

**Comment by jh-RLI** [comment]
On TOEP you can simply create an account if you want to :)

File: api/views.py [hunk 1]
```diff
@@ -453,10 +453,21 @@ def put(self, request, schema, table):
 
     def validate_column_names(self, column_definitions):
         """Raise APIError if any column name is invalid"""
+        MAX_COL_NAME_LENGTH = 50
```
Comment (wingechr) [inline api/views.py:456]: I guess I would prefer to put those constants at the top of the file, not inside the function, but ok for now

**Review by wingechr — APPROVED** [review]

**Review by wingechr — APPROVED** [review]

**Comment by chrwm** [comment]
The tests don't seem to restart..

**Pull request merged**